spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestAvroScan.java

```diff
@@ -68,7 +68,7 @@ public static void stopSpark() {
 
   @Override
   protected void writeAndValidate(Schema schema) throws IOException {
-    File parent = temp.newFolder("avro");
+    File parent = new File(temp.toFile(), "avro");
```

Comment:
```suggestion
    File parent =temp.resolve("avro").toFile();
```

Reply:
temp.resolve() was throwing errors before, now it's not. ;)
Updating it everywhere

Reply:
it was throwing an error before because `temp` was not initialized, because it is only initialized when the test is a JUnit5 test (using `org.junit.jupiter.api.Test`)